mindsdb/integrations/handlers/monkeylearn_handler/requirements.txt

```diff
@@ -0,0 +1,2 @@
+monkeylearn
```

Comment:
Please pin the version of monkeylearn to e.g. `<3.7.0` so that the integration doesn't break if there is a sudden major version release.

Reply:
done